src/cascadia/TerminalControl/Resources/en-US/Resources.resw

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 

```

Comment:
how did we accept another change that broke this file???

Reply:
I thought any time we use VS's table thing, it _always_ rewrites the top part.

Reply:
It can't _always_ rewrite it, it just always adds back these spaces and spelling errors if they're missing. :)